src/main/java/Duke.java

```diff
@@ -1,10 +1,60 @@
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
+    	String lines = "--------------------------------------------"; 
         String logo = " ____        _        \n"
```

Comment:
Indentation here need to changed into 4 spaces to make indentations consistent in your code